packages/mux-player/src/icons/volume-low.svg

```diff
@@ -1,4 +1,4 @@
-<svg aria-hidden="true" aria-hidden="true" viewBox="0 0 18 18" slot="low">
```

Comment:
`aria-hidden="true" aria-hidden="true"`
when you *really* want to make sure that it's hidden from screen readers.